net/lp_rpc.proto

```diff
@@ -112,9 +112,10 @@ message Capabilities {
 
     Constraints constraints = 5;
 
-    // Non-binary general constraints.
+    // Non-binary constraints.
     message Constraints {
       string minVersion = 1;
+      map<uint32, CapabilityConstraints> PerCapability = 2;
```

Comment:
Yeaah I think this looks much better :)

Is the breaking change gonna be painful?

Reply:
One thing that surprises me is the lack of RPC validation for calling the GetOrchestrator endpoint 🤔.

Reply:
Okay I found the behavoir:

**Old orch new Gateway**

This situation is protected since the Gateway throws a `grpc: error unmarshalling request: proto:\u00a0cannot parse invalid wire-format data` error during discovery so will not be considered 👍🏻. This behavoir happens [here](https://github.com/livepeer/go-livepeer/blob/f44238233b09424bb1fde2e1133d5bd08a23b925/net/lp_rpc_grpc.pb.go#L50).

**New Orch old Gateway**